docs/installation-and-operation/observability-with-prometheus.md

```diff
@@ -0,0 +1,158 @@
+---
+title: observability-with-prometheus
+---
+
+# Observability with Prometheus
+
+You can export [Prometheus](https://prometheus.io/) stats from your Metabase.
```

Comment:
Formally, that is not a "prometheus stats", that is metrics in a Prometheus [format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md). Not only Prometheus supports it, same as not only Java JMX tools support JMX format.